src/Repetitions.java

```diff
@@ -2,12 +2,12 @@
     public static void main(String[] args) {
 
         for (int i = 1; i <= 100; i++) {
-            if ((i % 3 == 0) && !(i % 5 == 0)) {
+            if ((i % 3 == 0) && (i % 5 == 0)) {
+                System.out.print("FizzBuzz(" + i + ")! ");
+            } else if ((i % 3 == 0)) {
```

Comment:
die runden klammern brauchst du hier nicht doppelt - beim und ists noch okay für die visuelle trennung aber hier sinds nur unnötig ;) 
brauchen würdest du sie in Zeile 5 auch nicht, wenn du sie für bessere lesbarkeit behalten magst ist das allerdings okay :)